src/lib/FormGroupLayoutLabel.tsx

```diff
@@ -30,33 +30,35 @@ const FormGroupLayoutLabel = <T extends FieldValues>(props: FormGroupLayoutLabel
   const checkboxLayout = layout === "checkbox";
 
   return (
-    <>
-      <Label check={checkboxLayout || switchLayout} for={fieldId}>
+    <div  onClick={(e) => e.stopPropagation()}>
```

Comment:
Hi @Harsh517-tech415 ,

First of all, thank you for your PR.

Wrapping tne entire label in a div will break the layout of check/switch input type. You should revert this change.
A possible solution could be to apply the stop propagation on the label instead.